File: workspace/8_math1/1.py
#https://www.acmicpc.net/problem/1712
#손익분기점
#A = Fixed, B = Per Cost, C = Sales
a,b,c = map(int,(input().split()))
if ( b >= c):
    print(-1)
else:
    print(int(a/(c-b)) + 1)

#문제를 잘 읽자... 쉬운건데 인풋 순서 착각해서 오래 걸림..

File: workspace/15_DP1/1.cpp
// https://www.acmicpc.net/problem/1003
//피보나치 함수

#include <iostream>
using namespace std;
long long fibArray[41] = {0,1,};

int fib(int n)
{
    int i = 0;
    for (i = 2; i <= n; i++)
    {
        fibArray[i] = fibArray[i - 1] + fibArray[i - 2];
    }

    return fibArray[n];
}

int main()
{

    int t, n;
    cin >> t;

    for (int i = 0; i < t; i++)
    {

        cin >> n;
        if (n == 0)
        {
            cout << "1 0" << '\n';
        }
        else
        {
            fib(n);
            cout << fibArray[n-1] << ' ' << fibArray[n] << '\n';
        }
    }

    return 0;
}